Assets/Scripts/Gameplay/GameplayObjects/Character/ClientCharacterVisualization.cs

```diff
@@ -43,11 +45,13 @@ public class ClientCharacterVisualization : NetworkBehaviour
 
         PhysicsWrapper m_PhysicsWrapper;
 
-        public bool CanPerformActions => m_NetState.CanPerformActions;
+        public bool CanPerformActions => m_ServerCharacter.CanPerformActions;
+
+        ServerCharacter m_ServerCharacter;
```

Comment:
We have ClientCharacterVisualization that points to ServerCharacter and vice-versa, wouldn't it be more symmetrical if ClientCharacterVisualization was renamed "ClientCharacter"?

Reply:
This way we'd have client that points to server and server that points to client?

Reply:
You're right.